.github/workflows/run_iree_dispatch_profiler.yml

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Workflow for running IREE Dispatch Profiler
+# (https://github.com/openxla/iree/tree/main/experimental/dispatch_profiler).
+#
+# IREE Dispatch Profiler validates functional correctness and performs profiling
+# on matmul, conv, and other ops on IREE CUDA and other back-ends.
+#
+# It uploads results to this a publicly accessible GCS bucket:
+# https://storage.googleapis.com/dispatch-profiler-artifacts
```

Comment:
I was modeling this after a handful of similar buckets that are also not fully-qualified: `mmperf-benchmark-artifacts`, `convperf-benchmark-artifacts`, `comparative-benchmark-artifacts`. I omitted the `iree-` prefix, because it's not clear that this will be solely focused on IREE in the long term. I could add it, but we'll have to re-create the bucket and fix the DB imports that were already added.

Reply:
I don't have much of an opinion beyond "names should be specific where possible". We might want to organize buckets for resource tracking / auditing / access control though? @GMNGeoffrey ?

Reply:
Bucket names are globally unique (seems like a design mistake, but ok): https://cloud.google.com/storage/docs/buckets#naming. So yes it's good to namespace them. Some other buckets we have:

```
iree-benchmark-artifacts/
iree-dana-db-archive/
iree-datasets/
iree-github-actions-jax-testsuite-artifacts/
iree-github-actions-postsubmit-artifacts/
iree-github-actions-presubmit-artifacts/
iree-model-artifacts/
iree-model-artifacts-archive/
iree-oss-build-badges/
iree-sccache/
iree-shared-files/
```